sdcm/utils/common.py

```diff
@@ -547,11 +547,20 @@ def clean_cloud_instances(tags_dict):
     """
     Remove all instances with specific tags from both AWS/GCE
 
-    :param tags_dict: a dict of the tag to select the instances, e.x. {"TestId": "9bc6879f-b1ef-47e1-99ab-020810aedbcc"}
+    :param tags_dict: a dict of the tag to select the instances,
+        e.x. {"TestId": ("9bc6879f-b1ef-47e1-99ab-020810aedbcc", "9bc6879f-b1ef-47e1-99ab-020810aedbcc"),
+              "RunByUser": "jenkins"}
     :return: None
     """
-    clean_instances_aws(tags_dict)
-    clean_instances_gce(tags_dict)
+    test_ids = tags_dict.pop("TestId", False)
+    if test_ids:
```

Comment:
using multiple types for the same variable is wrong, whether create a new function or address this specifically in the code that calls `clean_cloud_instances` (better)

Reply:
I would rather see the loop in `sct.py::clean_resources()`

```
    params = dict()
    if not (user or test_id):
       click.echo(clean_resources.get_help(ctx))

    if user:
        params['RunByUser'] = user

    if not test_id:
        clean_cloud_instances(params)
        click.echo('cleaned instances for {}'.format(params))

    for _test_id in test_id:
        params['TestId'] = _test_id
    
        clean_cloud_instances(params)
        click.echo('cleaned instances for {}'.format(params))
```